zenoh-plugin-ros2dds/src/lib.rs

```diff
@@ -79,7 +80,11 @@ macro_rules! ke_for_sure {
 
 lazy_static::lazy_static!(
     pub static ref VERSION_JSON_VALUE: Value =
-        serde_json::Value::String(ROS2Plugin::PLUGIN_LONG_VERSION.to_owned()).as_str().into();
+        serde_json::Value::String(ROS2Plugin::PLUGIN_LONG_VERSION.to_owned())
+        .as_str()
+        .map(|x| ZBytes::from(x))
+        .map(|z_bytes| Value::new(z_bytes,Encoding::default()))
```

Comment:
Value::from(value)?

Reply:
Implementing 
```rust
impl<T> From<T> for Value
where
    T: Into<ZBytes>,
{
    fn from(t: T) -> Self {
        Value::new(t.into(), encoding::Encoding::default())
    }
}
``` 
Conflicts with 
```rust

impl TryFrom<ZBytes> for Value {
    type Error = ZError;

    fn try_from(value: ZBytes) -> Result<Self, Self::Error> {
        ZSerde.deserialize(&value)
    }
}
```

So doing it this longer way is how we can avoid handling an extra `Result`.

Reply:
Make sense